bin/cli.js

```diff
@@ -148,7 +149,24 @@ const setRequestHeaders = () => {
   addUserAgentHeader('HubSpot CLI', pkg.version);
 };
 
+const NO_CONFIG_VALIDATION = {
+  init: { skip: true },
+  auth: { skip: true },
+};
+
 const loadConfigMiddleware = async options => {
+  const maybeValidateConfig = () => {
+    const shouldValidate = options._.every(
```

Comment:
Nice, I like this better. Just one more thought on it. I don't think `options._.every()` is doing anything for us here anymore. If we ever want to skip validation for something nested like `hs account auth` we'll probably have to switch to a different iteration pattern (maybe something recursive?). 

Also as it is now, we run the risk of accidentally opting certain commands out of validation. For example, if we added `list: {skip: true}` to the list with the intent of skipping validation for `hs list`. Using `options._.every()` here would also opt commands like `hs project list` out of validation.

Is `options._` guaranteed to exist? You could avoid accidentally triggering for other commands by simplifying to something like `if (NO_CONFIG_VALIDATION[options._[0]])`

Reply:
Ignore my comment here. I just noticed your work on this in https://github.com/HubSpot/hubspot-cli/pull/1310/files#diff-ca900686fca4680d4692bf587dc5da1322879c344688c187b5511cda27902c18R153